src/Symfony/Component/Security/Core/Tests/Authentication/Token/AbstractTokenTest.php

```diff
@@ -96,6 +97,19 @@ public function testSerialize()
         $this->assertEquals($token->getAttributes(), $uToken->getAttributes());
     }
 
+    public function testSerializeWithRoleObjects()
```

Comment:
@ro0NL I agree with you on this comment, I will remove all of these tests and just let one into `AbstractTokenTest` to avoid adding a lot of tests. Thanks